packages/pipeline/src/scripts/scrape_data.ts

```diff
@@ -220,6 +250,37 @@ function _scrapePriceToDB(timestamp: number, token: any): any {
     };
 }
 
+function _scrapeHistoricalPricesToDB(token: any, fromTimestamp: number, toTimestamp: number): any {
+    return (cb: () => void) => {
+        scrapeDataScripts.getHistoricalPriceSeries(token, BASE_SYMBOL, fromTimestamp, toTimestamp)
+            .then((data: any) => {
+                const parsedHistFullPrices: any = []
+                for(const histFullPrice of data['Data']) {
+                    const parsedHistFullPrice = (
+                        typeConverters.convertLogHistFullPriceToHistFullPriceObject(histFullPrice));
```

Comment:
Why is this call in parens?

Reply:
Let me know if you have a preference for long code lines; agree this wasn't consistent.